R/apply.scaling.R

```diff
@@ -1,15 +1,14 @@
 apply.scaling <- function(data.matrices, scaling.factors) {
 
-	# data.matrices can be a single matrix or a list of matrices
-	# if the data is a single matrix then the class will be 'matrix'	
-	if (class(data.matrices)[1] == 'matrix') {
-	
+	# `data.matrices` can be either 1. a single matrix or data frame, or 2. a list of matrices or data frames
+	# when the data is a single matrix or a data.frame
```

Comment:
**question: (non-blocking)**
> when the data is a single matrix or a data.frame

This comment seems redundant since it only reiterates the logic in the `if` block below. Can we remove it or do you want to comment on how the data will be treated depending on the class type?

Reply:
I can remove that!